lib/tutorials/en_US/auth.md

```diff
@@ -64,94 +165,144 @@ const start = async () => {
 start();
 ```
 
-First, we define our `users` database, which is a simple object in this example. Then we define a validation function, which is a feature specific to [hapi-auth-basic](https://github.com/hapijs/hapi-auth-basic) and allows us to verify that the user has provided valid credentials.
+First, we define our `users` database, which is a simple object in this example. Then we define a validation function, which is a feature specific to [hapi-auth-basic](https://github.com/hapijs/hapi-auth-basic) and allows us to verify that the user has provided valid credentials. For this validation function, we use `Bcrypt` to compare the user provided password with the hashed password in our database.  
 
-Next, we register the plugin, which creates a scheme with the name of `basic`. This is done within the plugin via [server.auth.scheme()](/api#serverauthschemename-scheme).
+Next, we register the plugin, which creates a scheme with the name of `basic`. This is done within the plugin via [server.auth.scheme()](/api#-serverauthschemename-scheme).
 
-Once the plugin has been registered, we use [server.auth.strategy()](/api#serverauthstrategyname-scheme-mode-options) to create a strategy with the name of `simple` that refers to our scheme named `basic`. We also pass an options object that gets passed to the scheme and allows us to configure its behavior.
+Once the plugin has been registered, we use [server.auth.strategy()](/api#-serverauthstrategyname-scheme-mode-options) to create a strategy with the name of `simple` that refers to our scheme named `basic`. We also pass an options object that gets passed to the scheme and allows us to configure its behavior.
```

Comment:
The dash was for the other link to change, this one should be `/api#server.auth.strategy()`

Reply:
Sorry bout that, I must've misread it...fixing now

Reply:
no problem